common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -126,7 +126,7 @@ class NativeAuthMsalController : BaseNativeAuthController() {
     fun signInStart(parameters: SignInStartCommandParameters): SignInStartCommandResult {
         LogSession.logMethodCall(
             tag = TAG,
-            correlationId = parameters.getCorrelationId(),
+            correlationId = parameters.correlationId,
```

Comment:
I don't think this compiles/works. See comment above `public String getCorrelationId()` in `CommandParameters.java`

Reply:
Which raises the question: did you test this? I.e. validate that correlation IDs are correctly logged?

Reply:
It somehow works in my local.
![image](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/assets/111055878/b04ba7f0-83c1-459a-ad33-96a5bdd3d575)

But haven't check the consistent of correlation IDs of the flow.

Reply:
Has validated that correlation IDs are correctly logged.